src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -224,29 +248,37 @@ class BinaryCompactObject : public DomainCreator<3> {
+      RadiusOuterCube, RadiusOuterSphere, InitialRefinement, InitialGridPoints,
+      UseEquiangularMap, UseProjectiveMap, UseLogarithmicMapOuterSphericalShell,
+      AdditionToOuterLayerRadialRefinementLevel, TimeDependence>;
 
   static constexpr OptionString help{
-      "The BinaryCompactObject domain is a general domain for two compact \n"
-      "objects. The user must provide the inner and outer radii of the \n"
-      "spherical shells surrounding each of the two compact objects A and B. \n"
-      "The user must also provide the radius of the sphere that \n"
-      "circumscribes the cube containing both compact objects, and the \n"
-      "radius of the outer boundary. The options ExciseInteriorA and \n"
-      "ExciseInteriorB determine whether the layer-zero blocks are present \n"
-      "inside each compact object. If set to `true`, the domain will not \n"
-      "contain layer zero for that object. The user specifies XCoordObjectA \n"
-      "and XCoordObjectB, the x-coordinates of the locations of the centers \n"
-      "of each compact object. In these coordinates, the location for the \n"
-      "axis of rotation is x=0. ObjectA is located on the left and ObjectB \n"
-      "is located on the right. Please make sure that your choices of \n"
-      "x-coordinate locations are such that the resulting center of mass\n"
-      "is located at zero."};
+      "The BinaryCompactObject domain is a general domain for two compact "
+      "objects. The user must provide the inner and outer radii of the "
+      "spherical shells surrounding each of the two compact objects A and B. "
+      "The user must also provide the radius of the sphere that "
+      "circumscribes the cube containing both compact objects, and the "
+      "radius of the outer boundary. The options ExciseInteriorA and "
+      "ExciseInteriorB determine whether the layer-zero blocks are present "
+      "inside each compact object. If set to `true`, the domain will not "
+      "contain layer zero for that object. The user specifies XCoordObjectA "
+      "and XCoordObjectB, the x-coordinates of the locations of the centers "
+      "of each compact object. In these coordinates, the location for the "
+      "axis of rotation is x=0. ObjectA is located on the left and ObjectB "
+      "is located on the right. Please make sure that your choices of "
+      "x-coordinate locations are such that the resulting center of mass "
+      "is located at zero.\n\n"
+      "Two radial layers join the outer cube to the spherical outer boundary. "
+      "The first of these layers transitions from sphericity == 0.0 on the "
+      "inner boundary to sphericity == 1.0 on the outer boundary. The second "
+      "has sphericity == 1 (so either linear or logarithmic mapping can be "
+      "used in the radial direction), extends to the spherical outer boundary "
+      "of the domain, and has a radial refinement level of (InitialRefinement "
+      "+ "
```

Comment:
If another change is requested that you need to address, please combine this line with the next, but it's not worth the effort of changing and pushing alone

Reply:
If not now, I'll fix in my followup PR that will add radial refinement to the inner spheres